### DISCUSSION TITLE
rebind select next hunk to another key

### DISCUSSION THREAD

**Discussion by tg0h** [jesseduffield/lazygit#2073] [Q&A] [answered]
how do I rebind the select next hunk command to another key?
![Screenshot 2022-07-31 at 16 53 51 - Brave Browser@2x](https://user-images.githubusercontent.com/7954426/182018621-d0970036-21b2-477f-aa73-a3430cc9c757.jpg)

**Comment by DusanLesan** [comment, marked as answer]
It looks to me that we cannot map this only for main context. You would need to remap the `nextBlock` or `nextBlock-alt`(currently using `l` key, so that also works for next hunk). That would look like this in config.yml:
```
keybinding:
  universal:
    nextBlock-alt: 'g'
```

**Reply by tg0h** [reply]
thanks! I did not realize that next block could also be used for next hunk.
